src/TitleTests.php

```diff
@@ -294,20 +294,22 @@ function check_variant_url_slugs($request_slug, $query) {
         global $wpdb;
 
         // Look up variant URL slugs that match the unknown request slug
-        $post_ids = $wpdb->get_col($wpdb->prepare("
-			SELECT post_id
-			FROM $wpdb->postmeta
-			WHERE meta_key LIKE 'title_variants_%_url_slug'
-			AND meta_value = '%s'
-		", $request_slug), 0);
-		if (empty($post_ids)) {
+        $post_id = $wpdb->get_var($wpdb->prepare("
```

Comment:
I'm changing from an array of IDs (`get_col`) to looking for a single post ID (`get_var`).